nqeit-cross-engine/src/test/java/org/projectnessie/integtests/crossengine/ITCrossEngine.java

```diff
@@ -50,12 +50,18 @@
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class ITCrossEngine {
 
+  private static final String NAMESPACE = System.getProperty("nqeit.namespace", "db");
```

Comment:
typo - nesqueit

Reply:
it wasnt a typo but just using the same abbreviation that the gradle modules use, but OK adjusted